packages/moonstone/docs/virtual-list-scroller.md

```diff
@@ -8,26 +8,22 @@ This document describes VirtualList, VirtualGridList, and Scroller.
 
 ### Basic usage of VirtualList and VirtualGridList
 
-*   At least four props below are required to show a list properly.
+*   At least three props below are required to show a list properly.
```

Comment:
Should we have a discussion of `isDisabledItem`?

Reply:
Yes. I added the document related with `isItemDisabled`.